source/multigrid/mg_transfer_matrix_free.cc

```diff
@@ -188,6 +188,22 @@ MGTransferMatrixFree<dim, Number>::prolongate(
   Assert((to_level >= 1) && (to_level <= level_dof_indices.size()),
          ExcIndexRange(to_level, 1, level_dof_indices.size() + 1));
 
+  dst = 0;
```

Comment:
I think you can delete the assertion two lines above because it is inside the `prolongate_and_add` function below.